cli-subcommands/create-container.js

```diff
@@ -103,7 +101,7 @@ exports.run = function(probo) {
 
 function exitWithError(message) {
   console.error(message);
-  process.exit(1);
```

Comment:
Actually, we do want to call process.exit here to exit non-zero.  We'll need to remove the es-lint rule that's preventing this.